README.md

```diff
@@ -36,6 +36,19 @@ You probably want to include the default stylesheet, too. It displays error mess
 
 If you're not a fan of the default styling, you can drop in your own stylesheet. The only new element to worry about is `vjs-errors-dialog` which is the container for the error messages.
 
+When using via **NPM/Browserify**
+```shell
+npm install videojs-errors
+```
+
+You need to add videojs-errors as plugin to the videojs module:
+```js
+import videojs from 'video.js';
+import videojsErrors from 'videojs-errors';
+
+videojs.plugin('errors', videojsErrors);
```

Comment:
This will throw an error if you're using Video.js 6 - you cannot re-register a plugin without de-registering the existing one first. Generally, plugins will register themselves, so doing `import 'videojs-errors';` should be sufficient.